.github/workflows/ci.yml

```diff
@@ -20,9 +20,6 @@ jobs:
         python-version:
           - 3.8
         toxenv:
-          - django22
-          - django30
-          - django31
```

Comment:
Dropping testing for these concurrently with the 3.2 upgrade deployment is probably fine for an infrequently changing repo like this one, but I'd make it a separate step on something like edx-platform where it's entirely possible that a new PR could introduce incompatibilities with 2.2 between the time the upgrade PR merges and determining if a rollback from stage or prod is needed.

Reply:
I will add these steps back.